static/img/c2c/taxii/example.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
As this is a JSON file, it should be moved to the `static/files` folder (currently it's in the images folder).

Reply:
You are correct Kim, it should be under static/files. But when we did the C2C cleanup, we moved all these examples to images folder by mistake. 
I will take this as a separate task and will move all the existing .json and .tf examples from img folder to  files folder.

Reply:
https://github.com/SumoLogic/sumologic-documentation/pull/3388#discussion_r1423053696